config_models/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.4'
+__version__ = '0.1.5'
```

Comment:
Given that you only updated requirements files, do you really need to bump the version again?  I don't think they're even included in the distributed package.